app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
+_cache = {}
```

Comment:
If you use global variables, you should call them using the capitalize mode . For example: CACHE_STORAGE = {}